.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
What do all of these editorconfig values do?

Reply:
They set the preference to not use `this.`. It's only a partial representation of the preferences for this repo, but it's a start. The reason I added these specific settings is to override my global machine settings which prefer `this.` I can remove it from this PR if preferred. In fact, I will, since it's an unrelated change.